specification/desktopvirtualization/DesktopVirtualization.Management/SessionHost.tsp

```diff
@@ -0,0 +1,115 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+import "./HostPool.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.DesktopVirtualization;
+/**
+ * Represents a SessionHost definition.
+ */
+@parentResource(HostPool)
+model SessionHost
+  is Azure.ResourceManager.ProxyResource<SessionHostProperties> {
+  ...ResourceNameParameter<
+    Resource = SessionHost,
+    KeyName = "sessionHostName",
+    SegmentName = "sessionHosts",
+    NamePattern = "^[A-Za-z0-9@.\\-_ ]*$"
+  >;
+}
+
+@armResourceOperations
+interface SessionHosts {
+  /**
+   * Get a session host.
+   */
+  get is ArmResourceRead<SessionHost>;
+
+  /**
+   * Update a session host.
+   */
+  @parameterVisibility
+  update is ArmCustomPatchSync<
```

Comment:
This should be changed to:

```tsp
update is ArmCustomPatchSync<
    SessionHost,
    SessionHostPatch,
    Request - {
      /**
       * The resource properties to be updated
       */
      @bodyRoot sessionHost?: SessionHostPatch;
      **
       * Force flag to update assign, unassign or reassign personal desktop.
       */
      @query("force")
      force?: boolean;
    }
>;
```

This will make the force parameter come after the request body